File: examples/babylonian/celsius-to-fahrenheit.js
// <Example :name="warm" celsius="30" />
// <Example :name="very cold" celsius="-30" />
// <Example :name="cold" celsius="-10" />
function celsiusToFahrenheit(celsius) {
    // <Probe :expression="celsius * Math.random()" />
    // <Assertion :expression="2 == 1 + 1" />
    // <Assertion :example="cold" :expected="14" />
    return celsius * 9/5 + 32;
}

File: examples/babylonian/celsius-to-fahrenheit.py
# <Example :name="cold" celsius="-10" />
# <Example :name="warm" celsius="30" />
def celsiusToFahrenheit(celsius):
    # <Probe :expression="[celsius] * 2" />
    # <Assertion :example="warm" :expression="result == 86" />
    result = celsius * 9/5 + 32
    return result
